src/components/BooksList.js

```diff
@@ -0,0 +1,52 @@
+// import { useEffect } from 'react';
```

Comment:
Kindly remove all commented code 👍 to make your code cleaner and professional

Please check for other occurrences 🙂